x-pack/plugins/apm/server/routes/settings/agent_configuration/find_exact_configuration.ts

```diff
@@ -51,5 +52,15 @@ export async function findExactConfiguration({
     return;
   }
 
-  return convertConfigSettingsToString(hit);
+  const configsAppliedToAgentsThroughFleet =
+    await getConfigsAppliedToAgentsThroughFleet({ setup });
+
+  return {
+    id: hit._id,
```

Comment:
is `id` new? is it a breaking change?

Reply:
This is new but it's not a breaking change as this function is only used in private APIs and in the function body of some public ones without changing the return type